source/aggregation-pipeline-builder.txt

```diff
@@ -40,6 +40,10 @@ You can:
 
 - :ref:`View or change pipeline settings <change-pipeline-settings>`
 
+- :ref:`Explain Plan for Pipelines <explain-plan-pipeline>` 
```

Comment:
I think we should lead with a verb here to match the rest of the bullets in the list.

```suggestion
- :ref:`View the explain plan for pipelines <explain-plan-pipeline>` 
```

Reply:
Done.